packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -490,7 +490,10 @@ abstract class Widget extends DiagnosticableTree {
 ///
 /// {@tool sample}
 ///
-/// The following is a skeleton of a stateless widget subclass called `GreenFrog`:
+/// The following is a skeleton of a stateless widget subclass called `GreenFrog`.
+///
+/// Normally, widgets have more constructor arguments, each of which corresponds
```

Comment:
[Sorry to be pedantic.] Constructors can have named parameters. When a constructor is called the values of those parameters are called the constructor's arguments. So all that to say: arguments => parameters here. Note also: it's very unlikely that we're consistent about this terminology in the rest of the API doc.